banyand/measure/query.go

```diff
@@ -378,20 +379,32 @@ func (qr *queryResult) Pull() *pbv1.MeasureResult {
 		if len(qr.data) == 0 {
 			return nil
 		}
-		// TODO:// Parallel load
-		tmpBlock := generateBlock()
-		defer releaseBlock(tmpBlock)
+		blankCursorList := []int{}
+		var mu sync.Mutex
+		var wg sync.WaitGroup
 		for i := 0; i < len(qr.data); i++ {
-			if !qr.data[i].loadData(tmpBlock) {
-				qr.data = append(qr.data[:i], qr.data[i+1:]...)
-				i--
-			}
-			if i < 0 {
-				continue
-			}
-			if qr.orderByTimestampDesc() {
-				qr.data[i].idx = len(qr.data[i].timestamps) - 1
-			}
+			wg.Add(1)
+			go func(i int) {
```

Comment:
Please limit the maximum number of goroutines to twice the number of CPU cores specified by the GOMAXPROCS. You can find more information about GOMAXPROCS [here](https://pkg.go.dev/runtime#GOMAXPROCS).